package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "scripts": {
     "build:demos": "node ./scripts/run.js prepublish demos",
-    "checkdeps": "node ./scripts/dependencies.test.js",
+    "checkdeps": "node ./scripts/check.dependencies.js --auto-fix",
```

Comment:
not sure it's a good idea. it could autofix on CI but it would stay on CI only then